### PR TITLE
Add a dropdown menu/component to the navbar

### PR DESCRIPTION
**Pull request by olatundeee**
Add a dropdown menu/component to the navbar with options to copy the current page url to the clipboard and also to navigate to the current url on the clipboard

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,6 +89,7 @@
     "react-icons": "^3.10.0",
     "react-jw-player": "^1.19.1",
     "react-markdown": "^4.3.1",
-    "react-router-dom": "^5.2.0"
+    "react-router-dom": "^5.2.0",
+    "reactjs-dropdown-component": "^1.1.2"
```
Comment (vaultec81) [inline package.json]: Unused dependancy?

File: src/renderer/components/TopNavbar.js [hunk 2]
```diff
@@ -109,6 +110,10 @@ class TopNavbar extends React.Component {
             window.history.back()
         }
 
+        function goToClip() {
+            navigator.clipboard.readText().then(clipText => {window.location.replace(clipText)});
```
Comment (vaultec81) [inline src/renderer/components/TopNavbar.js:114]: Make sure to only use the location hash, not the full path.

**Review by vaultec81 — CHANGES_REQUESTED** [review]
Looks good for the most part. I don't see anything out of the ordinary. Merge waiting on the changes I requested, and a visual confirmation on my end.

File: package.json (hunk 1 above)
Comment (olatundeee) [reply package.json]: Removed

File: src/renderer/components/TopNavbar.js (hunk 2 above)
Comment (olatundeee) [reply src/renderer/components/TopNavbar.js:114]: That is what I used from the onset, refer to the `windowLocationUrl` variable

**Pull request merged**